### ISSUE TITLE
logo for neatnet

### ISSUE THREAD

**Issue by anastassiavybornova** [uscuni/neatnet#153]
ask karo <3

**Comment by martinfleis** [comment]
@anastassiavybornova anything happened on this front? We can release without, geopandas had no logo for about 8 years :)

**Comment by martinfleis** [comment]
Finances?

**Comment by anastassiavybornova** [comment]
@martinfleis  yes, this might become a little official job for a graphic designer i've worked with together in the past

@martinfleis @jGaboardi do you have any inputs/requests/a wishlist of what the logo should/shouldn't be? my initial idea was to ask for something that:
* reminds the viewer of street networks (at least if contextualized as such)
* is rather coarse then fine grained (in terms of levels of detail)
* does not contain any text (or do we want a version which says "neatnet" in it?)
* and i would give some logo examples (geopandas, momepy, cityseer, networkx) as benchmark
any additions?

**Comment by martinfleis** [comment]
You got some funding for that? We can easily live forever without a logo...

> do you have any inputs/requests/a wishlist of what the logo should/shouldn't be?

your list is good.

**Comment by jGaboardi** [comment]
+1 with both Martin's comments.

**Comment by anastassiavybornova** [comment]
passed the instructions on, will be soon back with updates i hope
